telemetry/definitions/commonDefinitions.json

```
@@ -773,7 +773,44 @@
             "name": "codecatalyst_devEnvironmentWorkflowStep",
             "type": "string",
             "description": "Workflow step name"
+        },
+        {
+            "name": "authConnectionsCount",
+            "type": "int",
+            "description": "Current number of authentication connections the user has."
+        },
+        {
+            "name": "authUIElement",
+            "type": "string",
+            "description": "Feature tab user is interacting in.",
+            "allowedValues": [
+                "awsResource",
+                "codewhisperer",
+                "codecatalyst"
+            ]
+        },
+        {
+            "name": "unlockedTabs",
+            "type": "string",
+            "description": "comma delimtied list of tabs that user already has valid connections to."
```

```suggestion
            "description": "Comma delimited list of tabs that user already has valid connections to"
```

Maybe add an example as well?